minio/minio
# Purge the objects immediately when `x-minio-force-delete` header is passed in DeleteObject and DeleteBucket API

## Description

This PR is a continuation of https://github.com/minio/minio/pull/14818

If  header - `x-minio-force-delete` is passed in the DeleteObject and DeleteBucket API, the objects
will be purged immediately and free up the space

## Motivation and Context

this is a feature to immediately purge the object while deleting and freeing up the space from `.minio.sys/tmp/trash`

## How to test this PR?

Pass `x-minio-force-delete` headers in DeleteObject and DeleteBucket API - This should purge the trash entries immediately

Use the PR https://github.com/minio/minio-go/pull/1667 to test DeleteObject and DeleteBucket APIs with force delete option.

## Types of changes

## Checklist:
